common/app/views/mainAMP.scala.html

```diff
@@ -40,6 +42,22 @@
 
             @body
 
+            <div class="fc-container__inner">
+                <div class="fc-container__header__title">
+                    <span>popular</span>
+                </div>
+                <amp-list width="600" height="1000" layout="responsive" src="@if(Play.isDev) {https://www.thegulocal.com} else {@Configuration.ajax.url}/most-read-amp.json">
+                    <template type="amp-mustache">
+                        <div class="headline-list__item">
+                            <p class="headline-list__count">{{index}}</p>
+                            <h2 class="fc-item__title">
+                                <a href="{{url}}" class="fc-item__link">{{headline}}</a>
```

Comment:
haha, now I suddenly realised why it's called mustache :stuck_out_tongue_closed_eyes: